app/views/govuk_publishing_components/components/docs/select.yml

```
@@ -143,3 +143,15 @@ examples:
         value: 'option2'
       - text: 'Option three'
         value: 'option3'
+  with_page_heading:
+    description: This adds a H1 element with a label element inside containing the text supplied.
+    data:
+      id: "select-with-heading"
+      label: "This is a page heading"
```

Nitpick - there's a mix of single and double quotes, which (looking at the rest of the file) should be single quotes:

```suggestion
      id: 'select-with-heading'
      label: 'This is a page heading'
```

---

Thanks @injms - Nitpick resolved 👍 and rebased to master